### DISCUSSION TITLE
关于 label 的那些事

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#1363] [提问] [answered]
话说 PCL2 的仓库是如何做到让 Bug/优化/新反馈 3种 label 只有边框没有中间底色的

**Comment by LTCatt** [comment, marked as answer]
设置一个很淡的底色……你可以看这个：
![image](https://user-images.githubusercontent.com/40348278/221402329-1b9b9c47-338b-4568-ba05-9c6456f6e989.png)

**Reply by MoYuan-CN** [reply]
没用啊,改了之后就改变底色了

**Reply by WForst-Breeze** [reply]
您再试一下……？可以借助 Photoshop 等工具来辅助您确定其十六进制代码
![image](https://user-images.githubusercontent.com/110760354/222731758-4964039a-5609-46dd-b80d-ff6aeb0a635a.png)
![image](https://user-images.githubusercontent.com/110760354/222732201-58a3072c-ffa6-46ba-abb0-42bf13efe895.png)

**Reply by RawDiamondMC** [reply]
啊？原来这玩意有颜色？
![全都是白色.jpg](https://user-images.githubusercontent.com/112449001/222867425-147d8bcd-bad3-4064-a31c-f486daa9d307.png)
全都白的。。。

**Reply by LTCatt** [reply]
你开夜间模式就没颜色了 =。=

**Reply by RawDiamondMC** [reply]
> 你开夜间模式就没颜色了 =。=

草，算了，还是比较习惯夜间

**Comment by MoYuan-CN** [comment]
原来如此，好了
